### PR TITLE
HW37 Aliaksandr

### PR DESCRIPTION
**Pull request by AlexShcherba26**

### CODE REVIEW

File: hw_37/src/Tabs.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    active?: boolean | undefined;
+    disabled?: boolean
+};
+
+export const Tabs = ({ tabsArray }: { tabsArray: TabsProps[] }) => {
+    if (!tabsArray.length) {
+        return null;
+    }
+    return (<ul className="tabs">
+        {tabsArray.reduce((acc: any[] = [], arr) => {
```
Comment (jathpr) [inline hw_37/src/Tabs.tsx:14]: here length of tabsArray and the number of results elements are the same
prefer to use map, it's easy to read

Comment (jathpr) [reply hw_37/src/Tabs.tsx:14]: and avoid to use any

**Review by jathpr — APPROVED** [review]
nice work!

File: hw_37/src/Tabs.tsx (hunk 1 above)
Comment (AlexShcherba26) [reply hw_37/src/Tabs.tsx:14]: changed reduce method to map

File: hw_37/src/App.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { Title } from './Title';
+import { Menu } from './Menu';
+import { Tabs } from './Tabs';
+
+import './css/app.css';
+
+export const App = () => (
+  <div className='container'>
+    <Title></Title>
+    <Menu></Menu>
```
[the remark this answers is not in the document]
Comment (AlexShcherba26) [reply hw_37/src/App.tsx:12]: done

**Pull request merged**